docs/release-notes.rst

```diff
@@ -6,6 +6,48 @@
  Release Notes
 ###############
 
+**************
+ Version 0.28
+**************
+
+Version 0.28.0
+==============
+
+**Release Date:** February 05, 2024
+
+**Breaking Changes**
+
+-  Authentication: In the enterprise edition, in the master configuration, the
+   ``oidc.groups_claim_name`` setting that is used to set the string value of the authenticator's
+   claim name for groups has been changed to ``oidc.groups_attribute_name``. Similarly, the
+   ``oidc.display_name_claim_name`` setting that is used to set the user's display name in
+   Determined has been changed to ``oidc.display_name_attribute_name``.
+
+**New Features**
+
+-  Experiments: Add ``resources.is_single_node`` option which disallows scheduling the trial across
+   multiple nodes or pods, and forces it to be scheduled within a single container. If the requested
+   ``slots_per_trial`` count is impossible to fulfill in the cluster, the experiment submission will
+   be rejected.
```

Comment:
```suggestion
-  Experiments: Add ``resources.is_single_node`` option, which forces trials to be scheduled within
   single containers rather than across multiple nodes or pods. If the requested ``slots_per_trial``
   count is impossible to fulfill in the cluster, the experiment submission will be rejected.
```

Reply:
This is totally better language. Thanks.